### PR TITLE
Add aspenmesh role to webfuse

### PR DESCRIPTION
**Pull request by mukrishn**

### CODE REVIEW

File: ansible/roles/aspenmesh-servicemesh-setup/files/cacerts.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  root-cert.pem: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUJtVENDQVVDZ0F3SUJBZ0lVV01IQWh0anZOYkhpcGw0TkRPczBRVElaVkhnd0NnWUlLb1pJemowRUF3SXcKS3pFVE1CRUdBMVVFQ2d3S1FYTndaVzRnVFdWemFERVVNQklHQTFVRUF3d0xVbTl2ZENCRFFTQmxZMk13SGhjTgpNakV3TWpJME1Ua3pNelU1V2hjTk16RXdNakl5TVRrek16VTVXakFyTVJNd0VRWURWUVFLREFwQmMzQmxiaUJOClpYTm9NUlF3RWdZRFZRUUREQXRTYjI5MElFTkJJR1ZqWXpCWk1CTUdCeXFHU000OUFnRUdDQ3FHU000OUF3RUgKQTBJQUJES3FMeWhWUkNnRitSbUoyTU4rREFjcktoblRKN2c3MzdKUEJXcFdkeVB4Zjgrb3d0c2V2TndmK1ZuUwo0YWRXbk56YkpYY1dVcklMWTBkbUg3L2tOT1NqUWpCQU1CMEdBMVVkRGdRV0JCVDF1bU5yQXIrSUFsdWRoQXVpCjB1VHVObUo2ckRBUEJnTlZIUk1CQWY4RUJUQURBUUgvTUE0R0ExVWREd0VCL3dRRUF3SUM1REFLQmdncWhrak8KUFFRREFnTkhBREJFQWlCRHJUYktiZDRINXozdCtXK2dtMXRrMnlyS1h2WHp4NUxENHVaTytaL2tTd0lnQ3pFUwppS3dkK3M0NVJEMTZrbFVQTnk4dnhPRjFDRWpvMjdkSDhqaUdERzA9Ci0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0K
```
Comment (mukrishn) [inline ansible/roles/aspenmesh-servicemesh-setup/files/cacerts.yml:7]: FYI, these are all self signed sample CA files, required for aspenmesh deployment.

**Review by smalleni — APPROVED** [review]

**Comment by smalleni** [comment]
@mukrishn quick questions so u r using volumes from the ontap setup instead of local-storage or similar right? When you tested this did you face any issues because of using the ontap simulator or did they work just fine.

**Comment by mukrishn** [comment]
> @mukrishn quick questions so u r using volumes from the ontap setup instead of local-storage or similar right? When you tested this did you face any issues because of using the ontap simulator or did they work just fine.

@smalleni, @wilsondav and I tested ontap simulator and corrected few configs(already embedded in to golden image) to make sure PV can be claimed by pods. so it just worked fine for Aspenmesh pods as well.

**Comment by wilsondav** [comment]
> > @mukrishn quick questions so u r using volumes from the ontap setup instead of local-storage or similar right? When you tested this did you face any issues because of using the ontap simulator or did they work just fine.
> 
> @smalleni, @wilsondav and I tested ontap simulator and corrected few configs(already embedded in to golden image) to make sure PV can be claimed by pods. so it just worked fine for Aspenmesh pods as well.



> > @mukrishn quick questions so u r using volumes from the ontap setup instead of local-storage or similar right? When you tested this did you face any issues because of using the ontap simulator or did they work just fine.
> 
> @smalleni, @wilsondav and I tested ontap simulator and corrected few configs(already embedded in to golden image) to make sure PV can be claimed by pods. so it just worked fine for Aspenmesh pods as well.

@mukrishn @smalleni , we also need to add 2 more changes to the image to enable scaling to 500 volumes

* volume modify -vserver trident_test -volume svm_root -files 2048
* snap delete -vserver trident_test -volume svm_root *
* add-disks -aggregate aggr0_ontap212_01 -diskcount 2  
* vol size -vserver ontap212-01 -volume vol0 +1.5g

**Comment by smalleni** [comment]
> * volume modify -vserver trident_test -volume svm_root -files 2048
> * snap delete -vserver trident_test -volume svm_root *
> * add-disks -aggregate aggr0_ontap212_01 -diskcount 2
> * vol size -vserver ontap212-01 -volume vol0 +1.5g

Sweet, maybe we can update the gold image with that. merging this PR now.

**Pull request merged**